cmd/duffle/uninstall.go

```diff
@@ -3,71 +3,73 @@ package main
 import (
 	"errors"
 	"fmt"
+	"io"
 
 	"github.com/spf13/cobra"
 
 	"github.com/deislabs/duffle/pkg/action"
 )
 
-const usage = `Uninstalls an installation of a CNAB bundle.
+const uninstallUsage = `Uninstalls an installation of a CNAB bundle.
 
 When using '--parameters' or '--set', the uninstall command will replace the old
 parameters with the new ones supplied (even if the new set is an empty set). If neither
 '--parameters' nor '--set' is passed, then the parameters used for 'duffle install' will
 be re-used.
 `
 
-var uninstallDriver string
+var BundleAndBundleFileError = errors.New("Both --bundle and --bundle-file flags cannot be set")
```

Comment:
Selecting this line not so much to comment on the line itself but as a convenient spot to raise the following question:

For this `uninstall` action, what are some scenarios that a user would supply a bundle (via either the `--bundle` or `--bundle-file` option) when uninstalling?  Or, putting it another way, how would this provided bundle change the uninstallation action?

It seems we'd enable a user to get into some wild situations wherein they'd install a bundle, say, using the wordpress bundle.json and then subsequently uninstall this installation with, intentionally or unintentionally, a different bundle.json (either via a different _version_ of the wordpress bundle.json or an entirely different bundle.json)

Reply:
That's a good question. This PR is not changing existing functionality. As of now, you are able to pass in a bundle with uninstall. This is simply a refactor for UX.
An end user may want to pass in a bundle to run an uninstall action bc the new bundle or bundle version may contain updated logic for the uninstall action. The default scenario for uninstall does not require a bundle argument. Rather, this is a way for the user to override the existing uninstall action.

Reply:
I see, that definitely helps me understand.  Thanks @michelleN ~